### PR TITLE
Specific layout information for product id and SKU  

### PR DESCRIPTION
**Pull request by abrarpathan19**
## Purpose of this pull request

This pull request (PR) will provide the information regarding Specific layout for product id and SKU.  

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.3/frontend-dev-guide/layouts/product-layouts.html

### CODE REVIEW

**Comment by devops-devdocs** [comment]
An admin must run tests on this PR before it can be merged.

File: src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md [hunk 1]
```diff
@@ -20,6 +20,8 @@ Layout file | Description
 `catalog_product_view_type_grouped.xml` | Layout from this file is applied to `grouped` product only
 `catalog_product_view_type_simple.xml` | Layout from this file is applied to `simple` product only
 `catalog_product_view_type_virtual.xml` | Layout from this file is applied to `virtual` product only
+`catalog_product_view_id_123.xml` | Layout from this file is applied to specific `product ID` product only
+`catalog_product_view_sku_AB01.xml` | Layout from this file is applied to specific `SKU` product only
```
Comment (dmytro-ch) [inline src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md:24]:
```suggestion
`catalog_product_view_sku_{sku}.xml` | Layout from this file is applied to the specific product by `SKU` value. E.g. `catalog_product_view_sku_24-WG080.xml`
```

File: src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md [hunk 2]
```diff
@@ -20,6 +20,8 @@ Layout file | Description
 `catalog_product_view_type_grouped.xml` | Layout from this file is applied to `grouped` product only
 `catalog_product_view_type_simple.xml` | Layout from this file is applied to `simple` product only
 `catalog_product_view_type_virtual.xml` | Layout from this file is applied to `virtual` product only
+`catalog_product_view_id_123.xml` | Layout from this file is applied to specific `product ID` product only
```
Comment (dmytro-ch) [inline src/guides/v2.3/frontend-dev-guide/layouts/product-layouts.md:23]:
```suggestion
`catalog_product_view_id_{id}.xml` | Layout from this file is applied to the specific product by `Entity ID` value. E.g. `catalog_product_view_id_45.xml`
```

**Comment by abrarpathan19** [comment]
Hello @dmytro-ch, 
I have commit your suggested changes. 
Thanks for the helping :)

**Comment by dmytro-ch** [comment]
Hello, @abrarpathan19, you probably didn't get my recommendation correctly. Sorry for the misunderstanding.
I just asked you to provide the additional details in the description block. Because currently, it does not reflect the table’s content.
I didn’t mean to copy my suggestion. :)

Thank you!

**Comment by abrarpathan19** [comment]
> Hello, @abrarpathan19, perhaps, you understood my recommendation incorrectly. Sorry for the misunderstanding.
> I just asked you to provide the additional details in the description block. Because currently, it does not reflect the table’s content.
> I didn’t mean to [copy my suggestion](https://github.com/magento/devdocs/pull/7111/commits/68d78e9b7027c169bc32c3e156d85154c86e073e). :)
> 
> Thank you!

Hi,

Thanks for the review, please let me know what to do now. 

Thanks

**Comment by dobooth** [comment]
running tests

**Pull request merged**